jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -359,25 +362,6 @@ public RequestContext getRequestContext(Request baseRequest) {
       this.requestSpecificEnvironment = VmApiProxyEnvironment.createFromHeaders(System.getenv(),
           metadataCache, this, VmRuntimeUtils.getApiServerAddress(), wallclockTimer,
           VmRuntimeUtils.ONE_DAY_IN_MILLIS, defaultEnvironment);
```

Comment:
What will happen to async code that is handled after response was sent and tries to use GAE API?